test/ext_enzyme/enzyme.jl

```diff
@@ -6,10 +6,7 @@ using Functors
 using FiniteDifferences
 using CUDA
 
-_make_zero(x::Union{Number,AbstractArray}) = zero(x)
-_make_zero(x) = x
-make_zero(model) = fmap(_make_zero, model)
-## make_differential(model) = fmapstructure(make_zero, model) # NOT SUPPORTED, See https://github.com/EnzymeAD/Enzyme.jl/issues/1329
+make_zero(model) = Flux._make_zero!(model)
```

Comment:
CI says "UndefVarError: `_make_zero!` not defined" because it's in an extension... but I think anyway this should be the non-mutating one from Enzyme?

```suggestion
using Enzyme: make_zero
```

Reply:
Merged to try out on CI...